File: 程式語言 V2答案檔/Python/305.py
def compute(input_list):
    if input_list[1] == 1:
        return input_list[0] + input_list[2]
    elif input_list[1] == 2:
        return input_list[0] * input_list[2]

input_list = []
for i in range(3):
    x = int(input())
    input_list.append(x)
print(compute(input_list))

File: Programming Language(ver.2)/C++/305.cpp
#include <iostream>
using namespace std;

int compute(int []);
int main()
{
    int i, input[3];
    for (i = 0; i < 3; i++)
        cin >> input[i];
    cout << compute(input);
    return (0);
}

int compute(int s[])
{
    if (s[1] == 1)
        return s[0] + s[2];
    else if (s[1] == 2)
        return s[0] * s[2];
}